### ISSUE TITLE
[travis] Can we use pre-installed Chrome?

### ISSUE THREAD

**Issue by wilsonpage** [gaia-components/fast-list#52]
In our `.travis.yml` it seems we're downloading Chrome binaries before running tests. According to (this article](http://swizec.com/blog/how-to-run-javascript-tests-in-chrome-on-travis/swizec/6647) we should be able to just use the binary preinstalled on Travis boxes.

This would mean we can remove remove `sudo` dependency and [migrate from legacy hardware](http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade).

**Comment by etiennesegonzac** [comment]
The pre installed one is too old sadly :/ it's Chrome 34 IIRC. And we need at least 36 (again, IIRC). 
Can't remember which feature is missing I think it was something like for...of

**Comment by wilsonpage** [comment]
Ah OK, thanks for that clarification :)

**Comment by wilsonpage** [comment]
I guess we can make the change once pre-installed version updates.

**Comment by etiennesegonzac** [comment]
Definitely!I remember trying out the modern instances to check if chrome was updated there and it was indeed much faster :)gaia-components/fast-list wrote:I guess we can make the change once pre-installed version updates.

—Reply to this email directly or view it on GitHub.

Sent Using Firefox OS
